src/components/House.js

```diff
@@ -189,9 +189,10 @@ class House extends React.Component {
     Howler.volume(0);
     window.clearInterval(this.gameIntervalID);
     window.setTimeout(() => {
-       alert("GAME OVER. Your house has become awful");
-       window.setTimeout(() => {this.props.setGameState('menu');}, 2000);
+       document.getElementById('House-Id').innerHTML = "GAME OVER.&#13;&#10; Your house has become awful.";
+       window.setTimeout(() => {this.props.setGameState('enterScore');}, 2000);
```

Comment:
I might make this 3 or 4 seconds. It felt a little fast. Maybe it should show up on the same screen as the game above or below it, though it's fine as is.